src/js/scope_events.js

```diff
@@ -113,9 +113,7 @@
 		fireEvent('change', handleNumber);
 
 		// If this is a standard handle movement, fire the end event.
-		if ( data.handleNumber !== undefined ) {
-			fireEvent('end', data.handleNumber);
-		}
+		fireEvent('end', data.handleNumber || 2);
```

Comment:
fyi no clue what the `2` does in either case, but the event does indeed fire now for drag-fixed! just what i needed.